README.md

```diff
@@ -278,45 +283,54 @@ Spokestack.onSuccess = e => {
 
 ### Methods
 
-| Method Name                | Description                                                                     | Method Values  | OS           |
-| -------------------------- | ------------------------------------------------------------------------------- | -------------  | --           |
-| Spokestack.initialize()    | Initialize the speech pipeline; required for all other methods                  |   | Android, iOS |
-| Spokestack.start()         | Starts the speech pipeline. The speech pipeline starts in the `deactivate` state. | | Android, iOS |
-| Spokestack.stop()          | Stops the speech pipeline                                                         | | Android, iOS |
-| Spokestack.activate()      | Manually activate the speech pipeline                                             | | Android, iOS |
-| Spokestack.deactivate()    | Manually deactivate the speech pipeline                                           | | Android, iOS |
-| Spokestack.synthesize({'input': string, 'format': int, 'voice': string})             | Request a URL to a audio file of the specified voice speaking the input | format [0: text, 1: ssml, 2: speechmarkdown], voice ["demo-male"] | iOS, Android          |
-| Spokestack.classify(utterance, {})    | Classify the utterance with an intent/slot natural language understanding model |  | iOS          |
+                                                           | Method Name                | Description                                                                     | Method Values  | OS           |
```

Comment:
Why the extra indentation? Seems to mess up github's preview.

Reply:
spacemacs layer formatting is fighting against me and winning :(